src/validaze/core.clj

```diff
@@ -442,7 +442,7 @@
 (defn- -list-validator [user-defined-refinements [refinements inner-type]]
   (let [validator (refinement-kwd->validator refinements inner-type)
         is-user-defined? (contains? user-defined-refinements inner-type)
-        validation-fn (fn [v] (and (sequential? v) (every? #(validator %1) v)))
+        validation-fn (fn [v] (and (sequential? v) (every? #(nil? (validator %1)) v)))
```

Comment:
was this broken before? Looks like we're checking for the opposite now?

Reply:
It was a regression in the previous PR. Fixed in this one (returned to original).